openspec/changes/archive/2026-04-10-post-elim-ranking-update/design.md

```
@@ -0,0 +1,307 @@
+f.FinTiebreak AS Tiebreak,
+
+-- Qualification score (3rd tiebreaker)
+i.IndScore AS QualScore,
+
+-- Match numbers for writing averages back
+f.FinMatchNo AS RealMatchNo,
+f2.FinMatchNo AS OppRealMatchNo
+```
+
+Remove from the ORDER BY (sorting moves to PHP):
+```sql
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)